### PR TITLE
PPS naming anonymous namespace

### PR DESCRIPTION
**Pull request by fabferro**
#### PR description:

This PR adds a name to the namespace containing some PPS Pixel constants.
See issue #38977

#### PR validation:

Private validation on data. No differences seen, as expected.

### CODE REVIEW

**Comment by ChrisMisan** [comment]
type ctpps, bug-fix

**Comment by ChrisMisan** [comment]
please test

File: CondFormats/PPSObjects/interface/CTPPSPixelIndices.h [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
 */
 
-namespace {
+namespace RPixValues {
```
Comment (tvami) [inline CondFormats/PPSObjects/interface/CTPPSPixelIndices.h:55]: "For namespaces use lower case, e.g. namespace edm."
2.7 in https://cms-sw.github.io/cms_coding_rules.html

Comment (fabferro) [reply CondFormats/PPSObjects/interface/CTPPSPixelIndices.h]: @tvami, I did not check and had 50% of doing right ;-)
Done.

Comment (tvami) [reply CondFormats/PPSObjects/interface/CTPPSPixelIndices.h]: @fabferro thanks! haha!
Please squash the two commits, thanks!

**Comment by tvami** [comment]
@fabferro please squash the two commits, thanks

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
+alca 
 * tests pass
 * PR is according to the description

**Comment by jpata** [comment]
+reconstruction
- on the reco side, it's trivial (accessing from a namespace rather than global scope)
- no changes

**Comment by emanueleusai** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**